back/engines/commercial/public_api/spec/acceptance/v2/reactions_spec.rb

```diff
@@ -53,17 +53,15 @@
       expect(json_response_body[:reactions].first.keys).to match_array(
         %i[
           id mode reactable_id reactable_type created_at updated_at
-          user_id post_id post_type project_id
+          user_id post_id project_id
```

Comment:
The public API could potentially be very complex to deal with. The problem is that if we make too many changes on the public API, we might break services of our customers that rely on this. I already felt uncomfortable with deleting the post_type, but I suspect it's fine.

Ideally, we should create a v3 and rename post to idea there. At the same time, the v2 would still need to support post. Depending on the implementation (for example with the serializers we usually use), I think this could become a bit of a challenge to expose the same relationship with two different names. So also here I thought perhaps it's not worth the effort.

Perhaps we could check with Koen on Thursday which changes are worth spending more time on (API support, revertible migrations...)?

Reply:
I agree it's not a priority. Let's discuss with Koen, indeed.